samples/wifi/shutdown/README.rst

```diff
@@ -28,6 +28,15 @@ The sample:
 #. Shuts down the Wi-Fi driver.
 #. Puts the nRF5340 SoC in the lowest possible power consumption mode.
 
+Shutdown mode
+*************
+
+The Wi-Fi chipset power state is automatically managed by the Wi-Fi driver, if the network interface is down the Wi-Fi chipset is put in `shutdown` mode.
+And if the network interface is up the Wi-Fi chipset is put in `active` mode.
+
+The `shutdown` here refers to de-asserting `buck_en` signal to the Wi-Fi chipset powering the Wi-Fi chipset down.
+See `the nRF70 Series power states`_ for more information.
```

Comment:
I think this needs to be moved up in the sample description @richabp  @krish2718 . We need to link the documentation when we first introduce the "shutdown" term. Note: in nRF70 Series HW terminology the Shutdown is a State, not a Mode.

My proposal at line 10:

"The Shutdown sample demonstrates how to put the Nordic Semiconductor's Wi-Fi® chipset in Shutdown state, where the device is completely powered OFF. See `the nRF70 Series power states`_ for more information on nRF70 Series power states."